website/cue/reference/releases/0.44.0.cue

```diff
@@ -0,0 +1,302 @@
+
+				#### Fixes
+
+				- Fix a panic in float subtraction that produces NaN values. (https://github.com/vectordotdev/vrl/pull/1186)
+				"""
+		},
+		{
+			type: "enhancement"
+			description: """
+				`aws_s3` source now logs when S3 objects are fetched. If ACKs are enabled, it also logs on delivery.
+				"""
+			contributors: ["fdamstra"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The file sink now supports any input event type that the configured encoding supports. It previously only supported log events.
+				"""
+			contributors: ["nionata"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The NATS sink can now parse comma separated urls.
+				"""
+			contributors: ["whatcouldbepizza"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `kubernetes_logs` source now sets a `user-agent` header when querying [k8s apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
+				"""
+			contributors: ["ganelo"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fix the HTTP config provider to correctly parse TOML provided by the given HTTP endpoint.
+				"""
+			contributors: ["PriceHiller"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The `log_to_metric` transformer tag key are now template-able which enables tags expansion.
+				"""
+			contributors: ["titaneric"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add VRL function `parse_dnstap` that can parse dnstap data and produce output in the same format as `dnstap` source.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Adds a `force_path_style` option to the `aws_s3` sink that allows users to configure virtual host style addressing. The value defaults to `true` to maintain existing behavior.
+				"""
+			contributors: ["sam6258"]
+		},
+		{
+			type: "fix"
+			description: """
+				Allow the `skip_unknown_fields` setting to be optional, thereby allowing use of the defaults provided by the ClickHouse server. Setting it to `true` will permit skipping unknown fields and `false` will make ClickHouse strict on what fields it accepts.
+				"""
+			contributors: ["PriceHiller"]
+		},
+		{
+			type: "fix"
+			description: """
+				Retry Kafka messages that error with a policy violation so messages are not lost.
+				"""
+			contributors: ["PriceHiller"]
+		},
+		{
+			type: "fix"
+			description: """
+				Changes the fingerprint for file sources to use uncompressed file content
+				as a source of truth when fingerprinting lines and checking
+				`ignored_header_bytes`. Previously this was using the compressed bytes. For now, only gzip compression is supported.
+				"""
+			contributors: ["roykim98"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `filter` transform now generates a more accurate config when generated via `vector generate` by using a comparison rather than an assignment.
```

Comment:
```suggestion
				The `filter` transform now generates a more accurate config when generated with `vector generate` by using a comparison rather than an assignment.
```